Client/Frontend/UIConstants.swift

```diff
@@ -52,6 +52,9 @@ public struct UIConstants {
     // Firefox Orange
     static let ControlTintColor = UIColor(red: 240.0 / 255, green: 105.0 / 255, blue: 31.0 / 255, alpha: 1)
 
+    // Passcode dot gray
+    static let PasscodeDotColor = UIColor(colorString: "4A4A4A")
```

Comment:
We use `UIConstants(rgb: <hex>)` elsewhere in `UIConstants.swift`, so I'd suggest using that just for consistency.